02_virtualization_and_containers/README.md

```diff
@@ -2,11 +2,10 @@
 
 Learning goals:
 
-- Understand the difference between virtualization and containers.
-- Have clarity on when and how to use virtual machines and/or containers.
-- Work with virtual machines (VirtualBox) and manage them with Vagrant.
-- Work with containers with Docker (and Singularity).
-- Know how to set up own containers tailored to your requirements.
+- Understand the difference between virtualization and containers and when to use what.
```

Comment:
I would refrain from using `understand` as a verb, as Bloom's taxonomy is anyway about different levels of _understanding_. Better:

```suggestion
- Name differences between virtualization and containers and name use cases for each.
```